nuxeo-core/nuxeo-core-convert/src/main/java/org/nuxeo/ecm/core/convert/extension/ConverterDescriptor.java

```diff
@@ -76,6 +79,15 @@
     @XNodeList(value = "conversionSteps/subconverter", type = ArrayList.class, componentType = String.class)
     protected List<String> subConverters = new ArrayList<>();
 
+    /**
+     * Returns if the converter is expected to make a conversion or just return the input if it is of the same mime type as the output.
```

Comment:
```suggestion
     * Returns whether the conversion should be bypassed if the input blob mime type equals the converter destination mime type.
```
WDYT? @ataillefer @kevinleturc

Reply:
Yep, sounds clearer.

Reply:
done